source/codegen/metadata/nidmm/attributes.py

```diff
@@ -1,248 +1,263 @@
-# -*- coding: utf-8 -*-
-# This file is generated from NI-DMM API metadata version 19.6.0d7
 attributes = {
+    1050002: {
+        'access': 'read-write',
+        'channel_based': False,
+        'name': 'RANGE_CHECK',
+        'resettable': False,
+        'type': 'ViBoolean'
+    },
+    1050003: {
+        'access': 'read-write',
+        'channel_based': False,
+        'name': 'QUERY_INSTRUMENT_STATUS',
+        'resettable': False,
+        'type': 'ViBoolean'
+    },
+    1050004: {
+        'access': 'read-write',
+        'channel_based': False,
+        'name': 'CACHE',
+        'resettable': False,
+        'type': 'ViBoolean'
+    },
     1050005: {
         'access': 'read-write',
         'channel_based': False,
-        'documentation': {
```

Comment:
This PR shows 'documentation` and `description` tags  are are removed. Is that intended to be part of this PR?

Reply:
the documentation tags are not required for the generation. hence they are removed